src/TraceEvent/TraceEvent.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
One step at a time.  Lets get 2.0 working, and then we can see what delta we need to make 1.6 work.

Can you describe the value of supporting 1.6?